### ISSUE TITLE
Manually Linked Files: Allow unlinking single files if multiple are linked to same episode

### ISSUE THREAD

**Issue by harshithmohan** [ShokoAnime/Shoko-WebUI#1108]
![image](https://github.com/user-attachments/assets/27012532-194d-475f-bfe3-9ce44e8d2d9c)

For example, if we try to unlink and episode here, both the files linked to the episode will be unlinked. We have no way of choosing which file to unlink. Maybe we can add a dropdown to select the file or even a modal.

**Comment by da3dsoul** [comment]
Just encountered this in baine's db. At minimum, it should show the file names

**Comment by harshithmohan** [comment]
I don't think this UI's even staying once we move completely over to provider based linking
